vars/runWatcher.groovy

```diff
@@ -76,3 +77,11 @@ def call(Map args = [:]) {
         log(level: 'WARN', text: "runWatcher: Didn't work")
     }
 }
+
+def getStatusMessage(parsed) {
+  def status = '🚀'
+  if (parsed?.watch_record.result?.input?.payload?.hits) {
```

Comment:
In which cases this is `false`? If so, the status will be 🚀 without any text.

Reply:
I'll use the `?` so we can only support two cases, `total > 0` or otherwise no failed tests